distributions/sar653_dist.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_sar653_1(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.sqrt(1-x**2)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return np.sqrt(.232)
```

Comment:
the idea here is to have some code that came from an equation based on your pdf, x_min and x_max. Where does .232 come from?